runtimes/minimal/ubi9-python-3.12/Dockerfile.cpu

```diff
@@ -74,8 +74,12 @@ WORKDIR /opt/app-root/bin
 
 # Install Python packages from requirements.txt
 COPY ${MINIMAL_SOURCE_CODE}/pylock.toml ./
+
 # Copy Elyra dependencies for air-gapped enviroment
-COPY ${MINIMAL_SOURCE_CODE}/utils ./utils/
+COPY --chown=1001 ${MINIMAL_SOURCE_CODE}/utils ./utils/
+# Download Elyra Bootstrapper
+RUN curl -L https://raw.githubusercontent.com/opendatahub-io/elyra/refs/tags/v4.3.1/elyra/kfp/bootstrapper.py \
+         -o ./utils/bootstrapper.py
```

Comment:
Lets also set the env var ELYRA_INSTALL_PACKAGES false in the dockerfile too, 
so it is set in the runtime image.

```
ENV ELYRA_INSTALL_PACKAGES="false"
```